docs/en/samples/nonroot_access.md

```diff
@@ -0,0 +1,31 @@
+## Running Example
+   
+**Create a non-root user**
+```
+$ groupadd -g 1201 fluid-user-1 && \
+useradd -u 1201 -g fluid-user-1 fluid-user-1
+```
+The above command creates a non-root user`fluid-user-1`
+
+**Create a directory that belongs to the user**
+```
+$ mkdir -p /mnt/nonroot/user1_data && \
+echo "This is fluid-user-1's data" > /mnt/nonroot/user1_data/data1 && \
+chown -R fluid-user-1:fluid-user-1 /mnt/nonroot/user1_data && \
+chmod -R 0750 /mnt/nonroot/user1_data
+```
+The above command creates a directory `user1_data` belonging to `fluid-user-1` in the `/mnt/nonroot` directory, We will use the `data1` file in the `user1_data` directory to simulate the data dedicated to `fluid-user-1`
```

Comment:
```suggestion
The above command creates a directory `user1_data` belonging to `fluid-user-1` in the `/mnt/nonroot` directory, We will use the `data1` file in the `user1_data` directory to represent the data owned by `fluid-user-1`
```
I think "represent" and  "owned by" may be better

Reply:
I think your suggestion is great，I will commit your suggestion.